jest.config.js

```diff
@@ -7,6 +7,6 @@ module.exports = {
   ...require('./.config/jest.config'),
   moduleNameMapper: {
```

Comment:
Maybe we could aslo add:

```js
  resetMocks: true,
  clearMocks: true,
  resetModules: true,
```

Is there a situation where we don't want these options to be true?

Reply:
I think this promotes a cleaner approach to the tests. Will add it!